tests/testthat/test-install-torsten.R

```diff
@@ -0,0 +1,157 @@
+## Adapted from test-install.R from the cmdstanr package
+
+context("install_torsten")
```

Comment:
[context](https://testthat.r-lib.org/reference/context.html) has been deprecated; this can be safely dropped.

Reply:
done